locales/en.yml

```diff
@@ -70,3 +70,22 @@ en:
       description: Discover the story of our Logo
       page_title: Logo
     title: Logo of Ruby Nord
+  talks:
+    description: |
+      Find all the past talks from our Apéros Ruby.
+      Whether you're a beginner or a confirmed developer, feel free to give a talk and share your experience with the community.
+    level:
+      all: all levels
```

Comment:
I'd say that `beginner` is for fundamental concepts / things that we're supposed to know, such as: 

- `What's the difference between class and instance variables`
- `Initiation to rails`

And that `all` is a new concept/gem that doesn't require a high level of expertise to get it (eg. administrate, chartkick, etc.).

Finally, for the middleman talk, I'd say it can be `all`. This is quite subjective :/
What do you think?